frontend/src/components/forms/profileForm/ProfileForm.tsx

```diff
@@ -8,16 +8,18 @@ import { useForm } from 'react-hook-form'
 import { z } from 'zod'
 import { useUser } from '@clerk/clerk-react'
 
-import MotionContainer from '@/components/animation/MotionContainer'
-import StepNavigation from '@/components/forms/formComponents/StepNavigation'
-import FormInput from '@/components/forms/formComponents/FormInput'
-import FormSelect from '@/components/forms/formComponents/FormSelect'
+import { MotionContainer } from '@/components/animation/MotionContainer'
 import {
   titleSelectOptions,
   genderSelectOptions,
   profileFormSteps,
 } from '@/lib/formOptions/profileFormOptions'
-import FormStepButtons from '@/components/forms/formComponents/FormStepButtons'
+import {
+  FormStepButtons,
+  FormInput,
+  FormSelect,
+  FormStepNavigation,
+} from '@/components/forms/formComponents/index'
```

Comment:
You don't need to specify you're importing from the index file. If you import from a directory, js will look for an index file to resolve that import.

```suggestion
} from '@/components/forms/formComponents'
```